### ISSUE TITLE
Reproducible crash

### ISSUE THREAD

**Issue by skmgoldin** [trufflesuite/ganache#90]
Hello! I have a reproducible crash.

Run a Ganache.
```
git clone git@github.com:skmgoldin/simple-token-sale.git
cd simple-token-sale
git checkout ganache-bug
```
Change the following lines so that they point to your Ganache:
1. https://github.com/skmgoldin/simple-token-sale/blob/6b2344fb9f93e13c28046fd763c5f5ef64c3e439/truffle.js#L18-L19
2. https://github.com/skmgoldin/simple-token-sale/blob/6b2344fb9f93e13c28046fd763c5f5ef64c3e439/test/utils.js#L18-L19
Then:
```
npm i
npm run test
```
And Ganache will crash. I'm talking to Ganache running on OSX from inside an Ubuntu VM in case that makes a difference.

PLATFORM: darwin
GANACHE VERSION: 1.0.0

EXCEPTION:
Error: Lost heartbeat from server process!
    at Timeout.setInterval [as _onTimeout] (/src/Services/Chain.js:86:30)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)

**Comment by skmgoldin** [comment]
I suspect this has something to do with the `evm_mine` RPC call.

**Comment by benjamincburns** [comment]
@skmgoldin this may be a unique bug, or it may be a dupe of #83 (I'll be commenting there in just a moment). If you test against current master and the problem is gone, it's a dupe. If the problem is still there, please comment back here with the new error text.

**Comment by benjamincburns** [comment]
Closing this in favor of trufflesuite/ganache-cli#417, as I'm fairly certain that this shares the same underlying cause.

Also do note that we've recently released v1.0.2, which eliminates the "heartbeat lost" crash and adds better error reporting.
